Entities/AHuman.h

```diff
@@ -368,6 +376,13 @@ ENTITYALLOCATION(AHuman)
     Attachable * GetBGLeg() const { return (Attachable *)m_pBGLeg; }
 
 
+	/// <summary>
+	/// Gets the BG foot attachable of this.
+	/// </summary>
+	/// <returns>A pointer to the BG foot attachable of this. Ownership is NOT transferred!</returns>
+	Attachable * GetBGFoot() const { return m_pBGLeg->GetFoot(); }
```

Comment:
You need safety checks here I think. If an actor loses their leg this'll probably crash.
Something like this should do:
`if (m_pBGLeg) { return m_pBGLeft->GetFoot() } else { return nullptr; }`
may need to return something else if nullptr doesn't work, I'm not sure.